test/e2e/data/infrastructure-outscale/cluster-template-kcp-remediation.yaml

```diff
@@ -129,7 +129,7 @@ spec:
           name: "{{ ds.meta_data.local_hostname }}"
           kubeletExtraArgs:
             cloud-provider: external
-            provider-id: osc://'{{ ds.meta_data.instance_id }}'
+            provider-id: aws://${OSC_SUBREGION_NAME}/'{{ ds.meta_data.instance_id }}'
```

Comment:
Same takes what the metadata server gives you

Reply:
@outscale-mdr CI passed and validation on cloudgouv passed